MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfSegmentedControlTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfSegmentedControl without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+ <Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary />
+            <ResourceDictionary>
+                <x:String x:Key="SfSegmentedControlTheme">CommonTheme</x:String>
+                <Color x:Key="SfSegmentedControlNormalStroke">Red</Color>
+                <Color x:Key="SfSegmentedControlNormalBackground">Green</Color>
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+	</tr>
+    
+    <tr>
+    <td>
+            SfSegmentedControlNormalStroke    
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of border in segmented control.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+    <tr>
+    <td>
+            SfSegmentedControlNormalTextColor     
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of text in segmented control.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalBackground      
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background color of segmented control.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlDisabledSegmentBackground       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background Color of segment when it is disabled. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlDisabledSegmentTextColor        
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of segment item text when it is disabled.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionBackground         
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background color of segment when it is selected.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionTextColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of text when it is selected. 
```

Comment:
Color of segmented items text when it is selected.

Reply:
Updated